include/graphblas/reference/blas3.hpp

```diff
@@ -1309,20 +1774,17 @@ namespace grb {
 			"called with an output matrix C that does not match the output domain "
 			"of the given multiplication operator"
 		);
-		static_assert( ( !(
-				std::is_same< InputType1, void >::value ||
-				std::is_same< InputType2, void >::value )
-			), "grb::eWiseApply (reference, matrix <- matrix x matrix, operator): "
-			"the operator version of eWiseApply cannot be used if either of the "
-			"input matrices is a pattern matrix (of type void)"
+		static_assert(
+			!std::is_void< OutputType >::value,
+			"grb::eWiseApply: the elementwise mxm cannot be used if the"
+			" output matrix is a pattern matrix (of type void)"
```

Comment:
isn't the description wrong here? The issue is if the output is non-void, while both inputs are void (in the operator case at least)